client/favorites.jsx

```diff
@@ -71,10 +66,9 @@ export default class Favorites extends React.Component {
     if (this.state.favoritesData) {
       const favoriteHotels = this.state.favoritesData.map((info, idx) => {
         return (
-          <div key={idx} className={`policies text-center ${idx}`}>{info.hotelName}<img onClick={this.handleRemove} className={`pl-3 mb-1 trash-icon`} width="35rem" src="./images/trash.png"></img></div>
+          <div key={idx}  className={this.state.removalId === this.state.favoritesData[idx].hotelId ? 'fade1': ''}>{info.hotelName}<img data-hotel-id={this.state.favoritesData[idx].hotelId} onClick={this.handleRemove} className={`pl-3 mb-1 trash-icon`} width="35rem" src="./images/trash.png"></img></div>
```

Comment:
Your `info` parameter is `this.state.favoritesData[idx]`. Just do `info.hotelId`